src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,47 @@
+package lotto.view;
+
+import lotto.constant.Rank;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public final class OutputView extends ConsoleView {
+
+    private static final String LOTTO_COUNT_MESSAGE = "%d개를 구매했습니다.";
+    private static final String RESULT_HEADER = "당첨 통계";
+    private static final String DIVISION_LINE = "---";
+    private static final String RESULT_FORMAT_WITH_BONUS = "%d개 일치, 보너스 볼 일치 (%s원) - %d개";
+    private static final String RESULT_FORMAT = "%d개 일치 (%s원) - %d개";
+    private static final String ROI_FORMAT = "총 수익률은 %.1f%%입니다.";
+    private static final String PRIZE_WITH_COMMA = "%,d";
+
+    public void displayLottos(List<String> purchasedLotto) {
+        println(String.format(LOTTO_COUNT_MESSAGE, purchasedLotto.size()));
+        purchasedLotto.forEach(this::println);
+        this.lineBreak();
+    }
+
+    public void displayResults(List<Rank> ranks, double roi) {
+        println(RESULT_HEADER);
+        println(DIVISION_LINE);
+
+        Rank[] reversedRanks = Rank.values();
+        Arrays.sort(reversedRanks, Collections.reverseOrder());
+
+        for (Rank rank : reversedRanks) {
+            if (rank != Rank.NONE) {
+                int count = (int) ranks.stream()
+                    .filter(rank::equals)
+                    .count();
+                String format = rank == Rank.SECOND ? RESULT_FORMAT_WITH_BONUS : RESULT_FORMAT;
+                println(String.format(format, rank.getMatchCount(), String.format(PRIZE_WITH_COMMA, rank.getPrize()), count));
+            }
+        }
```

Comment:
(이렇게 바꿀 수 있어요!)


```suggestion
        Rank[] reversedRanks = Rank.values();
        Arrays.stream(Rank.values())
          .sorted(Collections.reverseOrder())
          .filter(rank -> rank != Rank.NONE)
          .map(this::메서드_포맷팅)
          .forEach(System.out::println);
        
        private String 메서드_포맷팅(Rank rank){
            int count = (int) ranks.stream()
                    .filter(rank::equals)
                    .count();
            String format = rank == Rank.SECOND ? RESULT_FORMAT_WITH_BONUS : RESULT_FORMAT;
            return String.format(format, rank.getMatchCount(), String.format(PRIZE_WITH_COMMA, rank.getPrize()), count);
        }
```
for문에 if를 사용하는 경우는 대부분 streamAPI를 활용해서 변경할 수 있어요 :)

Reply:
이 부분의 변경이 조금 어려웠었는데, 직접 코드까지 알려주셔서 감사합니다! for안에 if가 있는 경우 Stream을 통해 풀어나갈 수 있네요!
이제 알아차렸는데 format을 삼항연산자를 통해 indent가 사실 3인 코드네요 ㅠㅠ.. 
감사합니다.